<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1382</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>510</y>
      <width>133</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>1081</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0" rowminimumheight="0">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>100</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="lblAnhGoc">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lblAnhXuLy">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnFaceRecognition">
       <property name="text">
        <string>Face Recognition</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btnChonAnh">
       <property name="text">
        <string>Chọn ảnh</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnFaceDetection">
       <property name="text">
        <string>Face Detection</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>510</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GAUSS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>560</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MEDIAN</string>
    </property>
   </widget>
   <widget class="QSlider" name="hsMedian">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>560</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>11</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>11</number>
    </property>
    <property name="sliderPosition">
     <number>11</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSlider" name="hsGauss">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>500</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1382</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="LocNhieu">
    <property name="title">
     <string>Lọc nhiễu</string>
    </property>
    <addaction name="actionGaussianBlur"/>
    <addaction name="actionmedianBlur"/>
   </widget>
   <widget class="QMenu" name="DoBienDoCanh">
    <property name="title">
     <string>Dò biên, dò cạnh</string>
    </property>
    <addaction name="actionCanny"/>
    <addaction name="actionLaplacian"/>
    <addaction name="actionSobelX"/>
    <addaction name="actionSobelY"/>
    <addaction name="actionSobelCombined"/>
    <addaction name="actionPrewitt"/>
   </widget>
   <addaction name="LocNhieu"/>
   <addaction name="DoBienDoCanh"/>
  </widget>
  <action name="actionCanny">
   <property name="text">
    <string>Canny</string>
   </property>
  </action>
  <action name="actionSobelX">
   <property name="text">
    <string>SobelX</string>
   </property>
  </action>
  <action name="actionSobelY">
   <property name="text">
    <string>SobelY</string>
   </property>
  </action>
  <action name="actionSobelCombined">
   <property name="text">
    <string>SobelCombined</string>
   </property>
  </action>
  <action name="actionPrewitt">
   <property name="text">
    <string>Prewitt</string>
   </property>
  </action>
  <action name="actionCanny_2">
   <property name="text">
    <string>Canny</string>
   </property>
  </action>
  <action name="actionLaplacian">
   <property name="text">
    <string>Laplacian</string>
   </property>
  </action>
  <action name="actionGaussianBlur">
   <property name="text">
    <string>GaussianBlur</string>
   </property>
  </action>
  <action name="actionmedianBlur">
   <property name="text">
    <string>medianBlur</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
